DeviceDetector.NET.Tests/DeviceDetectorTest.cs

```diff
@@ -181,7 +181,7 @@ public void TestParse(string fileNme)
             var parser = new YamlParser<List<DeviceDetectorFixture>>();
             var _fixtureData = parser.ParseFile($"{Utils.CurrentDirectory()}\\fixtures\\{fileNme}.yml");
 
-            foreach (var expected in _fixtureData)
+            Parallel.ForEach(_fixtureData, expected =>
```

Comment:
@TimIrwin26 , I would add BoundedParallel here: https://github.com/Ascentis/Infrastructure/blob/master/Ascentis.Infrastructure/Threading/BoundedParallel/BoundedParallel.cs
to prevent the potential problem we've seen with HR ESS where Parallel has the potential to exhaust the internal .NET thread pool

Reply:
Not sure I want to add the dependency just for a test.

Reply:
You are right. Didn't pay attention it was a test.